util/config.h

```diff
@@ -9,4 +9,7 @@ extern bool symexec_print_each_value;
 
 extern bool skip_smt;
 
+extern bool disable_poison_input;
```

Comment:
Is this what we want? i.e. a global switch?
Or is there a use case where each input has a potential different behavior?  I can't think of anything right away, just checking your thoughts.

Reply:
For souper, we don't have any use-case as of now for disabling poison/undef for each input. I'd say let's just keep the global switch as of now and then we can add the input-specific support later when we need it.

Reply:
I agree we should keep things as simple as possible until a use-case for the other thing emerges